src/doom/s_sound.c

```diff
@@ -390,12 +391,26 @@ static int S_AdjustSoundParams(mobj_t *listener, mobj_t *source,
     return (*vol > 0);
 }
 
+int clamp(int x)
```

Comment:
Should be static and named consistently with other functions:

    static int Clamp(int x)